docs/topics/multiplatform/multiplatform-dsl-reference.md

```diff
@@ -716,9 +730,19 @@ kotlin {
             }
         }
     }
+    //Alternatively:
+    //compilerOptions {
+        //allWarningsAsErrors.set(true)
+    //}
```

Comment:
Maybe uncomment it?

Reply:
I've nowadded it as a separate code block so it won't be a problem if someone copies the code block.